source/js/classes/crop-exif.js

```diff
@@ -365,9 +365,12 @@ crop.service('cropEXIF', [function () {
                 }
                 http = null;
             };
-            http.open('GET', img.src, true);
             http.responseType = 'arraybuffer';
-            http.send(null);
+            http.open('GET', img.src, true);
+            try {
+                http.send(null);
+            } catch(e) {
```

Comment:
You aren't catching anything?
I would suggest to do a catch or only do the request when a certain standard is met.